src/interface/src/app/map/map.component.html

```diff
@@ -202,33 +202,54 @@
   <!-- Display containers for leaflet maps (up to 4) -->
   <div class="maps-container">
     <!-- Actions bar above the maps -->
-    <div class="map-options-bar">
+    <div class="map-actions-bar">
+      <button mat-flat-button
+        class="create-plan-button"
+        (click)="showAreaCreationActionButtons=!showAreaCreationActionButtons">
+        Create a plan
+        <span class="arrow" [ngClass]="{right: !showAreaCreationActionButtons, left: showAreaCreationActionButtons}"></span>
+      </button>
+
       <!-- Draw a planning area -->
       <button mat-flat-button
-          class="draw-area-button"
-          (click)="onAreaCreationOptionChange(AreaCreationOption.DRAW)">
-          <mat-icon>edit</mat-icon> Draw an area
-        </button>
+        class="draw-area-button"
+        [class.is-hidden]="!showAreaCreationActionButtons"
```

Comment:
any reason not to use the angular [hidden] attribute for this? Or *ngIf?

Reply:
I didn't want to actually remove the buttons from the DOM :p and when I tried [hidden], it wasn't working so this was the workaround lol